Sources/ComposableArchitecture/Store.swift

```diff
@@ -314,6 +314,30 @@ public final class Store<State, Action> {
     #endif
   }
 
+  /// Scopes the store to one that exposes child state and actions.
+  ///
+  /// A version of ``scope(state:action:)-9iai9`` that exposes the child state along
+  /// with the child action for scoping actions.
+  ///
+  /// - Parameters:
+  ///   - toChildState: A function that transforms `State` into `ChildState`.
+  ///   - fromChildAction: A function that transforms `ChildState` and `ChildAction` into `Action`.
+  /// - Returns: A new store with its domain (state and action) transformed.
+  @_disfavoredOverload
+  @_spi(Internals) public func scope<ChildState, ChildAction>(
```

Comment:
I wonder if we would wanna use A different SPI label than `Internals`? This is more like an experimental feature? In case-path some reflection APIs were recently added, behind `Reflection` spi label IIRC. So perhaps `@_spi(AdditionalScoping)` or similar?